docs/deploying-to-azure.md

```diff
@@ -175,7 +183,6 @@ az postgres flexible-server execute \
     --querytext "select * from villain"
 ```
 
-
 If you want to access the Postgres database from your local machine, you need to give access to your local IP address.
```

Comment:
Do we actually need to do this? Seems it created a firewall rule for me automatically when I deployed the db?

Reply:
You're right, I got rid of it.